packages/vm/tests/api/state/accountExists.spec.ts

```diff
@@ -41,8 +41,8 @@ tape('correctly apply new account gas fee on pre-Spurious Dragon hardforks', asy
     value: BigInt(0),
   }
 
-  const result = await vm.runCall(runCallArgs)
-  t.equal(result.execResult.gasUsed, BigInt(53552), 'vm correctly applies new account gas price')
+  const result = await vm.evm.runCall(runCallArgs)
+  t.ok(result.execResult.gasUsed === BigInt(53552), 'vm correctly applies new account gas price')
```

Comment:
can you please change back to `t.equal` (same with on L91)

Reply:
reverted both